activiti-services/activiti-services-query/activiti-services-query-model/src/main/java/org/activiti/services/query/app/model/ProcessInstance.java

```diff
@@ -0,0 +1,126 @@
+/*
```

Comment:
@gmalanga Can you check that this class exists in master? because it looks like you are adding this class, but it should exist already.. Maybe it was moved to a different package?